eq/node.cpp

```diff
@@ -467,13 +469,24 @@ void Node::frameTasksFinish( const uint128_t&, const uint32_t frameNumber )
 
 EventOCommand Node::sendError( const uint32_t error )
 {
-    return getConfig()->sendError( Event::NODE_ERROR, Error( error, getID( )));
+    return getConfig()->sendError( EVENT_NODE_ERROR, Error( error, getID( )));
 }
 
-bool Node::processEvent( const Event& event )
+bool Node::processEvent( const EventType type, AxisEvent& event )
 {
-    ConfigEvent configEvent( event );
-    getConfig()->sendEvent( configEvent );
+    getConfig()->sendEvent( type ) << event;
```

Comment:
feels having sendEvent() part of node etc. to omit the superfluous getConfig()...